src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -528,7 +538,7 @@ public void startJenkins() throws Throwable {
         new StreamCopyThread(description.toString(), proc.getErrorStream(), System.err).start();
         int tries = 0;
         while (true) {
-            if (port == 0 && portFile != null && portFile.exists()) {
+            if (port == 0 && portFile != null && portFile.canRead()) {
```

Comment:
What is the practical difference here?

Reply:
The motivation for this change is simple, Jenkins-test-harness does not control the way portFile gets created, potentially it could be created with some security/access options, so I suggest this change, if we are sure that it always accessible for a read once created we could revert this change.

Reply:
> potentially it could be created with some security/access options

Why would that happen? Is this something you observed, or just speculated on?

Reply:
speculatively, I've not observed it. I don't mind and will revert it